cmd/manager/main.go

```diff
@@ -17,15 +17,15 @@ package main
 
 import (
 	"fmt"
-	"os"
-
 	"github.com/kudobuilder/kudo/pkg/apis"
 	"github.com/kudobuilder/kudo/pkg/controller/instance"
 	"github.com/kudobuilder/kudo/pkg/controller/operator"
 	"github.com/kudobuilder/kudo/pkg/controller/operatorversion"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
```

Comment:
This packages are meant for the `kudoctl` command, not the manager. There isn't an equivalent package for the manager (yet). For now, the best approach is to directly add the `KudoSyncPeriod` function here in `manager/main.go` like it's done with the `ENABLE_WEBHOOKS` env var.